src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ProjectDebugPropertyPage.xaml

```diff
@@ -36,6 +36,48 @@
     </StringProperty.ValueEditors>
   </StringProperty>
 
+  <StringProperty Name="StartArguments"
+                  DisplayName="ReadOnly .user file command line arguments"
+                  Description="Deprecated. This is only visible if there are command line arguments in the .user file to pass to the executable."
+                  ReadOnly="False">
```

Comment:
Why not `ReadOnly="True"` here?

Reply:
removed in last commit

Reply:
I don't think this should be removed. I think it should be added.

Currently, this property will be editable. Yet I don't think that's what we want, is it? We want to display the value here, and allow the user to migrate the value from .user to .json.

Reply:
Even if we set this ReadOnly="True", cps will mark it as read only because it thinks this is a common property.